core/src/epicli/data/common/ansible/playbooks/roles/opendistro_for_elasticsearch/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 host_ip: "{{ ansible_default_ipv4.address | default(ansible_all_ipv4_addresses[0]) }}"
```

Comment:
Let's keep this file the same as in develop branch. Please remove line 2 since this property is unused.

Reply:
Yeah, I already though so, removed it.